crates/bevy_render/src/texture/fallback_image.rs

```diff
@@ -0,0 +1,48 @@
+use crate::{render_resource::*, texture::DefaultImageSampler};
+use bevy_derive::Deref;
+use bevy_ecs::prelude::FromWorld;
+use bevy_math::Vec2;
+use wgpu::{Extent3d, TextureDimension, TextureFormat};
+
+use crate::{
+    prelude::Image,
+    renderer::{RenderDevice, RenderQueue},
+    texture::{BevyDefault, GpuImage, ImageSampler},
+};
+
+#[derive(Deref)]
+pub struct FallbackImage(GpuImage);
```

Comment:
When is this used? Doc links here will help us catch if and when this public type becomes dead code.

Reply:
Added docs!